src/sdk.ts

```diff
@@ -584,6 +589,32 @@ export class DefaultRelayingServices implements RelayingServices {
         return exchangeRate;
     }
 
+    async getPingResponse(): Promise<PingResponse> {
+        return this.relayProvider.relayClient.getPingResponse();
+    }
+
+    async getPartners(): Promise<string[]> {
+        const { feesReceiver, relayWorkerAddress } =
+            await this.relayProvider.relayClient.getPingResponse();
+        let partners: Array<{ beneficiary: string; share: string }> = [];
+        if (feesReceiver != relayWorkerAddress) {
+            try {
+                const collector = await new this.web3Instance.eth.Contract(
+                    Collector.abi,
+                    feesReceiver
+                );
+                console.log(collector);
```

Comment:
I'd use `logLevel` instead of using `console.log`

Reply:
updated